phoenix/slackbot/tasks.py

```diff
@@ -778,3 +780,33 @@ def postmortem_notifications():
         else:
             if solution.created < label_limit:
                 postmortem_label_notify(solution)
+
+
+def communication_assignee_should_be_notified(outage):
+    now = arrow.now()
+    last_notified = outage.communication_assignee_last_notified
+    if not last_notified:
+        # Before the first notification.
+        last_notified = outage.created
+    minutes_passed = divmod((now - last_notified).seconds, 60)[0]
+    return minutes_passed >= settings.NOTIFY_COMMUNICATION_ASSIGNEE_MINUTES
```

Comment:
Flip so we do `NOTIFY_COMMUNICATION_ASSIGNEE_MINUTES  * 60` for the check instead